rust/src/util.rs

```diff
@@ -15,6 +15,8 @@
  * 02110-1301, USA.
  */
 
+//! Utility module to test strings for valid UTF-8.
```

Comment:
```
Utility module.
```

So it doesn't get out of date. Yes, we appear to have a few of these, but that's probably a cleanup job for another task/day.